lib/client/redirect.go

```diff
@@ -336,12 +344,20 @@ func (rd *Redirector) wrapCallback(fn func(http.ResponseWriter, *http.Request) (
 			case rd.errorC <- err:
 			case <-rd.context.Done():
 			}
-			http.Redirect(w, r, errorURL, http.StatusFound)
+			redirectURL := errorURL
+			if keys.IsPrivateKeyPolicyError(err) {
+				redirectURL = terminalRedirectURL
+			}
```

Comment:
idea: For `hardware_key_touch`, `hardware_key`, and `hardware_key_touch_and_pin` it would be ideal if the redirect window automatically closed since the second pop up will open without further user interaction. I think we'd need to introduce another redirect URL that can just close the tab, if that's even possible?